deps/build.jl

```diff
@@ -9,7 +9,7 @@ else
 
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/windows/amd64/", os = :Windows)
     provides(Binaries, URI("http://bitbucket.org/MDukhan/yeppp/downloads/yeppp-$ver.tar.bz2"), libyepppdep, unpacked_dir = "yeppp-$ver/binaries/macosx/x86_64/", os = :Darwin)
-    @linux_only push!(BinDeps.defaults, Binaries)
+    @compat @static is_linux() ? push!(BinDeps.defaults, Binaries) : nothing
```

Comment:
@tkelman will the behaviour note change if we `pop!`, because in the original version I do not see any `pop!`. Thanks for the `Compat` suggestion, for this I assume this is the relevant [PR](https://github.com/JuliaLang/Compat.jl/pull/204), so the `Compat` version would be `v0.7.19`.